lib/pages/chat_details/chat_details_edit.dart

```diff
@@ -182,26 +185,46 @@ class ChatDetailsEditController extends State<ChatDetailsEdit>
   ) async {
     final result = await FilePicker.platform.pickFiles(
       type: FileType.image,
+      withData: false,
```

Comment:
IMO, by default it is false, we don't need to set a value for it.

Reply:
Removed